test/e2e/cypress/e2e/checks_customization.cy.js

```diff
@@ -1,5 +1,7 @@
 import * as checksSelectionPage from '../pageObject/checks_customization_po';
 
+const COROSYN_MAX_MESSAGES_CHECK_ID = '00081D';
+const CHECK_COROSYNC_TOKEN_TIMEOUT = '156F64';
```

Comment:
Not sure making a constant for check ids helps a lot here.

Anyway, If we intend to keep the constants let's make the naming pattern consistent

Reply:
I will pass the values directly to the functions then :)
I guess it is more readable :)